python_qt_binding/__init__.py

```diff
@@ -52,9 +52,13 @@
 """
 
 import sys
-from .binding_helper import loadUi, QT_BINDING, QT_BINDING_MODULES, QT_BINDING_VERSION  # @UnusedImport
 
-# register all binding modules as sub modules of this package (python_qt_binding) for easy importing
+from python_qt_binding.binding_helper import \
+    loadUi, QT_BINDING, QT_BINDING_MODULES, QT_BINDING_VERSION
+
+print('Qt5 Python Binding {}: Version {}'.format(QT_BINDING, QT_BINDING_VERSION))
```

Comment:
This message was probably used for debugging and should be removed again.

Reply:
This line was added, as well as the loadUI = loadUi so that flake8 would not complain about unused variables. Because the init exposes loadUI, QT_BINDING and QT_BINDING_VERSION, but doesn't use them in the file flake8 thinks they are unused. 

@dirk-thomas Do you have a recommended resolution?

Reply:
You can suppress a specific `flake8` warning by appending `  # noqa: X123` to the line.

Reply:
Adding noqa: F401 to suppress unused imports. I guess it's recommended to add these to an __all__ variable, but I'm not sure how that would work with the dynamic module imports associated with this package.